dynawo/sources/Models/CPP/ModelNetwork/DYNModelTwoWindingsTransformer.cpp

```diff
@@ -1412,13 +1405,13 @@ ModelTwoWindingsTransformer::evalG(const double& t) {
       vValue = modelBusMonitored_->getCurrentU(ModelBus::UType_);
       nodeOff = modelBusMonitored_->getSwitchOff();
     }
-    modelRatioChanger_->evalG(t, vValue, nodeOff, &g_[offset], disableInternalTapChanger_, tapChangerLocked_, getConnectionState() == CLOSED);
+    modelRatioChanger_->evalG(t, vValue, nodeOff, &g_[offset], disableInternalRatioTapChanger_, tapChangerLocked_, getConnectionState() == CLOSED);
```

Comment:
Wouldn't it be easier to remove the associated roots when they are disabled instead of recalculating them?